dm/master/config.go

```diff
@@ -324,6 +324,17 @@ func (c *Config) genEmbedEtcdConfig() (*embed.Config, error) {
 	cfg.InitialCluster = c.InitialCluster
 	cfg.ClusterState = c.InitialClusterState
 
+	// use zap as the logger for embed etcd
+	// NOTE: `genEmbedEtcdConfig` can only be called after logger initialized.
+	// NOTE: if using zap logger for etcd, must build it before any concurrent gRPC calls,
+	// otherwise, DATA RACE occur in builder and gRPC.
+	cfg.ZapLoggerBuilder = embed.NewZapCoreLoggerBuilder(log.L().Logger, log.L().Core(), log.Props().Syncer)
```

Comment:
can we add some tag for etcd's log, just like we add "sarama" in binlog https://github.com/pingcap/tidb-binlog/blob/3f0d148e33777caa1e721000999e84acaebefc28/pkg/util/log.go#L89

Reply:
added `[component="embed etcd"]` in https://github.com/pingcap/dm/pull/360/commits/709720aaf03dc57ca8a28a7f28ba38198e5bb7f7.